WORKSPACE

```diff
@@ -1,5 +1,3 @@
-workspace(name = "tensorflow_addons")
-
```

Comment:
Why do we get rid of the workspace name here?

Reply:
Yeah, so I tried for a while to get this to work. Windows seemed unwilling to copy the files during `build_pip_pkg` if the directory was specified using workspace. Possibly related to windows sym-links but I'm not really sure. It's worth figuring out, but I've had about as much of windows build as I can stand for a while :P

The workspace name is something I added during CUDA10.1 PR and isn't being done for custom-op:
https://github.com/tensorflow/custom-op/blob/master/WORKSPACE